integration-tests/kotlin/pom.xml

```diff
@@ -14,17 +14,6 @@
     <name>Quarkus - Integration Tests - Kotlin</name>
 
     <dependencies>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.checkerframework</groupId>
-                    <artifactId>checker-qual</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
```

Comment:
Removed Guava dependency of this module because `ImmutableMap.of()` can be easily replaced by `Map.of()`